arxiv-opensearch-cohere/README.md

```diff
@@ -0,0 +1,256 @@
+# Cohere / Opensearch implementation
+
+The following steps outline how a user would perform vector search with Cohere and Opensearch. Starting with spinning up a local OpenSearch instance and how a user would store Cohere embeddings in an OpenSearch index, and then explaining the methods for retrieval using these embeddings.
+Note, these instructions are based on OpenSearch v2.5.0.
+
+## Step 1: Spin up an instance of OpenSearch
+
+Run `docker-compose up` to spin up the Opensearch instance running at `http://localhost:9200`
```

Comment:
isn't the port 8080 exposed?

Reply:
this is for the opensearch server, the streamlit app would be at 8080